Update Opticks to support Optimizely's JS SDK v5.3.2 (#81)
FindHotel/opticks: README.md

At the heart of our experimentation framework is the `toggle` function.

> A toggle allow you to switch between multiple experiment variants (a/b/c/...)

Reviewer: ```suggestion
A toggle allows you to switch between multiple experiment variants (a/b/c/...)
```

## Toggles

> Toggles can be used to implement a/b/c style testing and on/off values as well.

Reviewer 2: ```suggestion
Toggles can be used to implement a/b/c style MVT testing and on/off feature flags as well.
```

> We specify multiple variants of which one is active at any time.

Reviewer 2: ```suggestion
We specify multiple variants of which only one is active at any time.
```
